assets/blocks/course-categories-block/course-categories.scss

```diff
@@ -10,16 +10,24 @@
 		white-space: nowrap;
 		margin-bottom: 5px;
 
-		&:not(:last-child) {
+		&:not( :last-child ) {
 			margin-right: 10px;
 		}
+
+		&:not( .has-background ) {
+			background-color: #1a1d20;
```

Comment:
I think we should use the color from the Figma design, which is #3858E9.

Reply:
Updated here bf62180412dfc25c917c0abc670ee4e597bf5e41